seml/config.py

```diff
@@ -189,7 +189,7 @@ def generate_configs(experiment_config):
 
         if len(redefined_parameters) > 0:
             logging.warning(f"Found redefined parameters in {current_sub_name}: {redefined_parameters}. "
-                            f"Redefinitions of parameters override earlier ones.")
+                            f"Definitions in sub-configs override higher (more general) ones.")
```

Comment:
@danielzuegner Is this strictly true? Imho it should be.

Reply:
Yes, it's true.